microprofile/openapi/src/test/java/io/helidon/microprofile/openapi/TestMultiJandex.java

```diff
@@ -22,11 +22,13 @@
 import org.jboss.jandex.ClassInfo;
 import org.jboss.jandex.DotName;
 import org.jboss.jandex.IndexView;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+@Disabled
```

Comment:
Just double checking in case you decide to use `CompositeIndex` above, or if it's relevant